scanapi/reporter.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+from jinja2 import Environment, PackageLoader, select_autoescape
+import logging
+
+from scanapi.settings import SETTINGS
+
+logger = logging.getLogger(__name__)
+
+
+class Reporter:
+    def __init__(self, docs_path, reporter, template):
```

Comment:
Maybe we can already change `docs_path` to `report_path` now too.

Reply:
I like the idea. The only point is that this will be a breaking change, since other projects inform `docs_path` in their yaml files. If you're ok with that, I can make the change 👍

Reply:
Or you're saying to keep `docs_path` outside and rename it to `report_path` only in this method?

Reply:
I am ok with that, I can update the project that is using the lib after the release, no problem :)

Reply:
@camilamaia what do you think of `output_path` as most of the other tools use? That way we also won't conflict `-r` for `reporter` and `report_path`. That would be a `-o`.

Reply:
Perfect!

Reply:
Done!